frappe/public/css/website.css

```diff
@@ -380,7 +380,7 @@ h3,
 h4,
 h5,
 h6 {
-  font-weight: 300;
+  font-weight: 400 !important;
```

Comment:
Why not 700? 300 is book, 400 is normal weight, 500 is semi-bold and 700 is bold